Frontend/CO.CDP.OrganisationApp/Pages/Shared/_FlashMessage.cshtml

```diff
@@ -35,6 +35,35 @@
         @{
             FlashMessage message = tempDataService.Get<FlashMessage>(FlashMessageTypes.Important)!;
         }
+        <div class="govuk-notification-banner__content">
+            <p class="govuk-notification-banner__heading">@Html.Raw(message.Heading)</p>
+
+            @if (message.Description != null)
+            {
+                <p>@Html.Raw(message.Description)</p>
+            }
+        </div>
+    </div>
+}
+
+@if (tempDataService.Peek<FlashMessage>(FlashMessageTypes.Failure) != null)
+{
+    FlashMessage message = tempDataService.Get<FlashMessage>(FlashMessageTypes.Failure)!;
+
+    <div class="govuk-notification-banner govuk-!-margin-top-8 app-notification-banner--failure" aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner">
+        <div class="govuk-notification-banner__header">
+            <h2 class="govuk-notification-banner__title" id="govuk-notification-banner-title">
+                @if (message.Title != null)
+                {
+                    @message.Title;
+                }
+                else
+                {
+                    @:Failure  
```

Comment:
This word "Failure" here needs localizing

Reply:
You could consider making `Supplementary_ContactUs_Failure_Heading` into `Global_Failure_Heading` and just using it here